fre/cmor/cmor_mixer.py

```diff
@@ -4,408 +4,383 @@
 see README.md for additional information on `fre cmor run` (cmor_mixer.py) usage
 '''
 
+import glob
 import json
 import logging
-fre_logger = logging.getLogger(__name__)
-
 import os
 from pathlib import Path
 import shutil
 import subprocess
 
+import cmor
 import numpy as np
-
 import netCDF4 as nc
-import cmor
+
 from .cmor_helpers import *
 
+fre_logger = logging.getLogger(__name__)
 
+# Constants
+ACCEPTED_VERT_DIMS = ["z_l", "landuse", "plev39", "plev30", "plev19", "plev8",
+                      "height2m", "level", "lev", "levhalf"]
+NON_HYBRID_SIGMA_COORDS = ["landuse", "plev39", "plev30", "plev19", "plev8", "height2m"]
+ALT_HYBRID_SIGMA_COORDS = ["level", "lev", "levhalf"]
+DEPTH_COORDS = ["z_l"]
```

Comment:
I like it. These settings have to be somewhere, and they shouldn't really be user-editable.